packages/frontend/src/navigation/Nav.tsx

```diff
@@ -68,6 +71,27 @@ export const Nav = (): JSX.Element => {
             </AdminOnly>
             <NavItem icon={faBook} description="Logs" to="/eventlogs" />
             <NavItem icon={faQuestionCircle} description="FAQ" to="/faq" />
+            <NavItem
```

Comment:
Place items at bottom of navbar and make them pink. I added a better wireframe pic to the issue. Also, adjust the light/dark switch.